src/main/java/org/calrissian/flowmix/api/aggregator/AbstractGrouplessAggregator.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2015 Calrissian.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.calrissian.flowmix.api.aggregator;
+
+import static java.lang.System.currentTimeMillis;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import static java.util.UUID.randomUUID;
+import org.apache.commons.lang.StringUtils;
+import org.calrissian.flowmix.api.Aggregator;
+import org.calrissian.flowmix.core.model.event.AggregatedEvent;
+import org.calrissian.flowmix.core.support.window.WindowItem;
+import org.calrissian.mango.domain.Tuple;
+import org.calrissian.mango.domain.event.BaseEvent;
+import org.calrissian.mango.domain.event.Event;
+
+/**
+ *
+ * This abstract Aggregator doesn't mantain the whole group, you need to add and
+ * evict the tuples with simple operations and emit them with complex ones.
+ * 
+ * Currently under discussion
+ *
+ * @author Miguel A. Fuentes Buchholtz
+ * @param <T> Aggregation result type
+ * @param <F> Field type
+ */
+@Deprecated
```

Comment:
> (sorry I think this the very first serious contribution to OS)

No problem at all. Your contributions are definitely welcomed. Generally the first several patches to an open source project help the contributor get in line with the processes and structure of the project.